trade-finance-manager-ui/server/helpers/premium-payments-table-checkbox-id-helper.ts

```diff
@@ -1,14 +1,32 @@
 import { CURRENCY, Currency, FEE_RECORD_STATUS, FeeRecordStatus } from '@ukef/dtfs2-common';
 import { PremiumPaymentsTableCheckboxId } from '../types/premium-payments-table-checkbox-id';
 
+/**
+ * Regular expression group to match comma-separated numeric IDs.
+ * This regex captures one or more digits followed by optional commas.
+ * Example matches: "1", "1,2", "1,2,3", etc.
+ */
+const COMMA_SEPARATED_IDS_REGEX_GROUP = `(?<commaSeparatedIds>(\\d+,?)+)`;
+const COMMA_SEPARATED_IDS_REGEX = new RegExp(COMMA_SEPARATED_IDS_REGEX_GROUP);
+
+/**
+ * Regular expression group to match valid currency codes.
+ * This regex captures any of the currency codes defined in the CURRENCY enum.
+ * Example matches: "GBP", "USD", etc.
+ */
 const CURRENCY_REGEX_GROUP = `(?<currency>${Object.values(CURRENCY).join('|')})`;
 const CURRENCY_REGEX = new RegExp(CURRENCY_REGEX_GROUP);
 
+/**
+ * Regular expression group to match valid fee record statuses.
+ * This regex captures any of the status values defined in the FEE_RECORD_STATUS enum.
+ * Example matches: "TO_DO", "DOES_NOT_MATCH", etc.
```

Comment:
TO_DO?

Reply:
This is a valid value for a fee record status